lib/src/main/java/com/shopify/checkoutsheetkit/CheckoutDialog.kt

```diff
@@ -50,6 +50,8 @@ internal class CheckoutDialog(
     context: Context,
 ) : Dialog(context) {
 
+    private var error: CheckoutException? = null
```

Comment:
How come we need to store this on the class? Are we resetting it back to null at some point?

Reply:
And the reason for placing the code in the cancel listener is because it catches swipes as well as close button clicks